cmd/operator-sdk/cli/legacy.go

```diff
@@ -73,11 +73,11 @@ func GetCLIRoot() *cobra.Command {
 		add.NewCmd(),
 		alpha.NewCmd(),
 		build.NewCmd(),
-		bundle.NewCmd(),
+		bundle.NewCmdLegacy(),
```

Comment:
May would be better to keep `bundle.NewCmd(),` and inside of it we identify if is the new layout or the old one as it has been done for the other cases.

Reply:
Well we've already centralized the check for layout in `main.go`, which will determine which commands are used. We shouldn't re-check layout in subcommands.

Reply:
I see now. (`cmd/operator-sdk/cli/legacy.go` x `cmd/operator-sdk/cli/cli.go`)